### PR TITLE
refactor(script): Refactor check-aur.py

### PR DESCRIPTION
**Pull request by 67au**

### CODE REVIEW

File: check-aur.py [hunk 1]
```diff
@@ -1,58 +1,62 @@
...
+            if r:
+                yay_dep_nums.append(n)
+
+    yaydeps_dir = repo_dir.parent.joinpath('.yaydeps')
+
+    if len(yay_dep_nums) > 0:
+        async with aiofiles.open(yaydeps_dir.joinpath(repo_dir.stem), 'w', encoding='utf-8') as f:
+            await f.write('\n'.join([*(deps[n] for n in sorted(yay_dep_nums)), '']))
+
+    return 1
```
Comment (KunoiSayami) [inline check-aur.py:55]: Should return 0 on normal exit.

**Review by KunoiSayami — CHANGES_REQUESTED** [review]

File: check-aur.py (hunk 1 above)
Comment (67au) [reply check-aur.py]: Updated.

**Review by KunoiSayami — APPROVED** [review]
LGTM

**Comment by KunoiSayami** [comment]
Thanks!

**Pull request merged**